internal/testutils/bubblewrap.go

```diff
@@ -0,0 +1,182 @@
+package testutils
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	bubbleWrapSupportsUnprivilegedNamespacesOnce sync.Once
+	bubbleWrapSupportsUnprivilegedNamespaces     bool
+
+	bubbleWrapNeedsSudoOnce sync.Once
+	bubbleWrapNeedsSudo     bool
+)
+
+// CanRunBubbleWrapTest checks whether we can run tests running in bubblewrap or
+// sip the tests otherwise.
```

Comment:
```suggestion
// skip the tests otherwise.
```

Reply:
Yah thanks for noticing, I had already a fixup with it that apparently I didn't push...